main.less

```diff
@@ -30,16 +25,24 @@
     border: 1px solid #deca7e;
 }
 
-.errors {
-    width: 25px;
-    cursor: pointer;
-}
+.hint-marker-positioning {
+  left: auto !important; /* CodeMirror will set this value, but we don't want it set! */
+  right: 1em;
+  background-color: transparent;
 
-.gutterCursor {
-    cursor: pointer;
+  margin: 0px;
+  padding: 0px;
```

Comment:
Doesn't matter.  If you touch a line of code, you own it :)  A reviewer will comment on any line of code that's in play.  The real way to look at this is that it is "our" code vs "my" code, and we all have to make it better.